
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://madanucd.github.io/gregor_linkml/">
      
      <link rel="icon" href="assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.0, mkdocs-material-8.5.10">
    
    
      
        <title>gregor_linkml</title>
      
    
    
      <link rel="stylesheet" href="assets/stylesheets/main.975780f9.min.css">
      
        
        <link rel="stylesheet" href="assets/stylesheets/palette.2505c338.min.css">
        
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL(".",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="None" data-md-color-accent="None">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#gregor_linkml" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="." title="gregor_linkml" class="md-header__button md-logo" aria-label="gregor_linkml" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            gregor_linkml
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Index
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/madanucd/gregor_linkml" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="." title="gregor_linkml" class="md-nav__button md-logo" aria-label="gregor_linkml" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    gregor_linkml
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/madanucd/gregor_linkml" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          Index
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="." class="md-nav__link md-nav__link--active">
        Index
      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    Classes
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    Slots
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    Enumerations
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    Types
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    Subsets
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="about/" class="md-nav__link">
        About
      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    Classes
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    Slots
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    Enumerations
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    Types
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    Subsets
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  <a href="https://github.com/madanucd/gregor_linkml/edit/master/docs/index.md" title="Edit this page" class="md-content__button md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>


<h1 id="gregor_linkml">gregor_linkml</h1>
<p>This repository includes the LinkML model and tooling for the GREGoR data model.</p>
<p>URI: https://w3id.org/madanucd/gregor_linkml</p>
<p>Name: gregor_linkml</p>
<h2 id="classes">Classes</h2>
<table>
<thead>
<tr>
<th>Class</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Aligned/">Aligned</a></td>
<td>Table containing aligned information</td>
</tr>
<tr>
<td><a href="AlignedAtacShortRead/">AlignedAtacShortRead</a></td>
<td>Table containing aligned_atac_short_read information</td>
</tr>
<tr>
<td><a href="AlignedDnaShortRead/">AlignedDnaShortRead</a></td>
<td>Table containing aligned_dna_short_read information</td>
</tr>
<tr>
<td><a href="AlignedDnaShortReadSet/">AlignedDnaShortReadSet</a></td>
<td>Table containing aligned_dna_short_read_set information</td>
</tr>
<tr>
<td><a href="AlignedNanopore/">AlignedNanopore</a></td>
<td>Table containing aligned_nanopore information</td>
</tr>
<tr>
<td><a href="AlignedNanoporeSet/">AlignedNanoporeSet</a></td>
<td>Table containing aligned_nanopore_set information</td>
</tr>
<tr>
<td><a href="AlignedPacBio/">AlignedPacBio</a></td>
<td>Table containing aligned_pac_bio information</td>
</tr>
<tr>
<td><a href="AlignedPacBioSet/">AlignedPacBioSet</a></td>
<td>Table containing aligned_pac_bio_set information</td>
</tr>
<tr>
<td><a href="AlignedRnaShortRead/">AlignedRnaShortRead</a></td>
<td>Table containing aligned_rna_short_read information</td>
</tr>
<tr>
<td><a href="AlleleSpecificAtacShortRead/">AlleleSpecificAtacShortRead</a></td>
<td>Table containing allele_specific_atac_short_read information</td>
</tr>
<tr>
<td><a href="Analyte/">Analyte</a></td>
<td>Table containing analyte information</td>
</tr>
<tr>
<td><a href="CalledPeaksAtacShortRead/">CalledPeaksAtacShortRead</a></td>
<td>Table containing called_peaks_atac_short_read information</td>
</tr>
<tr>
<td><a href="CalledVariantsDnaShortRead/">CalledVariantsDnaShortRead</a></td>
<td>Table containing called_variants_dna_short_read information</td>
</tr>
<tr>
<td><a href="CalledVariantsNanopore/">CalledVariantsNanopore</a></td>
<td>Table containing called_variants_nanopore information</td>
</tr>
<tr>
<td><a href="CalledVariantsPacBio/">CalledVariantsPacBio</a></td>
<td>Table containing called_variants_pac_bio information</td>
</tr>
<tr>
<td><a href="Experiment/">Experiment</a></td>
<td>Table containing experiment information</td>
</tr>
<tr>
<td><a href="ExperimentAtacShortRead/">ExperimentAtacShortRead</a></td>
<td>Table containing experiment_atac_short_read information</td>
</tr>
<tr>
<td><a href="ExperimentDnaShortRead/">ExperimentDnaShortRead</a></td>
<td>Table containing experiment_dna_short_read information</td>
</tr>
<tr>
<td><a href="ExperimentNanopore/">ExperimentNanopore</a></td>
<td>Table containing experiment_nanopore information</td>
</tr>
<tr>
<td><a href="ExperimentPacBio/">ExperimentPacBio</a></td>
<td>Table containing experiment_pac_bio information</td>
</tr>
<tr>
<td><a href="ExperimentRnaShortRead/">ExperimentRnaShortRead</a></td>
<td>Table containing experiment_rna_short_read information</td>
</tr>
<tr>
<td><a href="Family/">Family</a></td>
<td>Table containing family information</td>
</tr>
<tr>
<td><a href="GeneticFindings/">GeneticFindings</a></td>
<td>Table containing genetic_findings information</td>
</tr>
<tr>
<td><a href="Participant/">Participant</a></td>
<td>Table containing participant information</td>
</tr>
<tr>
<td><a href="Phenotype/">Phenotype</a></td>
<td>Table containing phenotype information</td>
</tr>
</tbody>
</table>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Slot</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="5prime3prime_bias/">5prime3prime_bias</a></td>
<td></td>
</tr>
<tr>
<td><a href="additional_details/">additional_details</a></td>
<td>modifier of a term where the additional details are not supported/available a...</td>
</tr>
<tr>
<td><a href="additional_family_members_with_variant/">additional_family_members_with_variant</a></td>
<td>List of related participant IDs carrying the same variant</td>
</tr>
<tr>
<td><a href="additional_modifiers/">additional_modifiers</a></td>
<td></td>
</tr>
<tr>
<td><a href="affected_status/">affected_status</a></td>
<td>Indicate affected status of individual (overall with respect to primary pheno...</td>
</tr>
<tr>
<td><a href="age_at_collection/">age_at_collection</a></td>
<td>age or participant in years at biosample collection</td>
</tr>
<tr>
<td><a href="age_at_enrollment/">age_at_enrollment</a></td>
<td>age in years at which consent was originally obtained</td>
</tr>
<tr>
<td><a href="age_at_last_observation/">age_at_last_observation</a></td>
<td>Age at last observation, aka age in years at the last time the center can vou...</td>
</tr>
<tr>
<td><a href="aligned_atac_short_read_file/">aligned_atac_short_read_file</a></td>
<td>name and path of file with aligned reads</td>
</tr>
<tr>
<td><a href="aligned_atac_short_read_id/">aligned_atac_short_read_id</a></td>
<td>identifier for aligned ATAC-seq data</td>
</tr>
<tr>
<td><a href="aligned_atac_short_read_index_file/">aligned_atac_short_read_index_file</a></td>
<td>name and path of index file corresponding to aligned reads file</td>
</tr>
<tr>
<td><a href="aligned_dna_short_read_file/">aligned_dna_short_read_file</a></td>
<td>name and path of file with aligned reads</td>
</tr>
<tr>
<td><a href="aligned_dna_short_read_id/">aligned_dna_short_read_id</a></td>
<td></td>
</tr>
<tr>
<td><a href="aligned_dna_short_read_index_file/">aligned_dna_short_read_index_file</a></td>
<td>name and path of index file corresponding to aligned reads file</td>
</tr>
<tr>
<td><a href="aligned_dna_short_read_set_id/">aligned_dna_short_read_set_id</a></td>
<td>identifier for experiment set</td>
</tr>
<tr>
<td><a href="aligned_file/">aligned_file</a></td>
<td></td>
</tr>
<tr>
<td><a href="aligned_id/">aligned_id</a></td>
<td>table_name</td>
</tr>
<tr>
<td><a href="aligned_index_file/">aligned_index_file</a></td>
<td></td>
</tr>
<tr>
<td><a href="aligned_nanopore_file/">aligned_nanopore_file</a></td>
<td>name and path of file with aligned reads</td>
</tr>
<tr>
<td><a href="aligned_nanopore_id/">aligned_nanopore_id</a></td>
<td></td>
</tr>
<tr>
<td><a href="aligned_nanopore_index_file/">aligned_nanopore_index_file</a></td>
<td>name and path of index file corresponding to aligned reads file</td>
</tr>
<tr>
<td><a href="aligned_nanopore_set_id/">aligned_nanopore_set_id</a></td>
<td>identifier for experiment set</td>
</tr>
<tr>
<td><a href="aligned_pac_bio_file/">aligned_pac_bio_file</a></td>
<td>name and path of file with aligned reads</td>
</tr>
<tr>
<td><a href="aligned_pac_bio_id/">aligned_pac_bio_id</a></td>
<td></td>
</tr>
<tr>
<td><a href="aligned_pac_bio_index_file/">aligned_pac_bio_index_file</a></td>
<td>name and path of index file corresponding to aligned reads file</td>
</tr>
<tr>
<td><a href="aligned_pac_bio_set_id/">aligned_pac_bio_set_id</a></td>
<td>identifier for experiment set</td>
</tr>
<tr>
<td><a href="aligned_read_length_mean/">aligned_read_length_mean</a></td>
<td>Mean length of aligned reads</td>
</tr>
<tr>
<td><a href="aligned_rna_short_read_file/">aligned_rna_short_read_file</a></td>
<td>name and path of file with aligned reads</td>
</tr>
<tr>
<td><a href="aligned_rna_short_read_id/">aligned_rna_short_read_id</a></td>
<td>identifier for aligned_short_read (primary key)</td>
</tr>
<tr>
<td><a href="aligned_rna_short_read_index_file/">aligned_rna_short_read_index_file</a></td>
<td>name and path of index file corresponding to aligned reads file</td>
</tr>
<tr>
<td><a href="alignment_log_file/">alignment_log_file</a></td>
<td>path of (log) file with all parameters for alignment software</td>
</tr>
<tr>
<td><a href="alignment_postprocessing/">alignment_postprocessing</a></td>
<td>If any post processing was applied</td>
</tr>
<tr>
<td><a href="alignment_software/">alignment_software</a></td>
<td>Software including version number</td>
</tr>
<tr>
<td><a href="allele_balance_or_heteroplasmy_percentage/">allele_balance_or_heteroplasmy_percentage</a></td>
<td>Reported allele balance (mosaic) or heteroplasmy percentage (mitochondrial)</td>
</tr>
<tr>
<td><a href="alt/">alt</a></td>
<td>Alternate position of the variant</td>
</tr>
<tr>
<td><a href="analysis_details/">analysis_details</a></td>
<td>brief description of the analysis pipeline used for producing the asc_file; p...</td>
</tr>
<tr>
<td><a href="analyte_id/">analyte_id</a></td>
<td></td>
</tr>
<tr>
<td><a href="analyte_processing_details/">analyte_processing_details</a></td>
<td>details about how the analyte or original biosample was extracted or processe...</td>
</tr>
<tr>
<td><a href="analyte_type/">analyte_type</a></td>
<td>analyte derived from the primary_biosample</td>
</tr>
<tr>
<td><a href="ancestry_detail/">ancestry_detail</a></td>
<td>Additional specific ancestry description free text beyond what is captured by...</td>
</tr>
<tr>
<td><a href="application_kit/">application_kit</a></td>
<td>Library prep kits for special applications</td>
</tr>
<tr>
<td><a href="asc_atac_short_read_id/">asc_atac_short_read_id</a></td>
<td>unique key for table (anvil requirement)</td>
</tr>
<tr>
<td><a href="asc_file/">asc_file</a></td>
<td>name and path of the tsv file with allele-specific chromatin accessibility me...</td>
</tr>
<tr>
<td><a href="asc_md5sum/">asc_md5sum</a></td>
<td>md5 checksum for called_peaks_file</td>
</tr>
<tr>
<td><a href="barcode_kit/">barcode_kit</a></td>
<td>Barcode kit used</td>
</tr>
<tr>
<td><a href="by_strand/">by_strand</a></td>
<td>run reports separate reads per strand</td>
</tr>
<tr>
<td><a href="called_peaks_atac_short_read_id/">called_peaks_atac_short_read_id</a></td>
<td>identifier for called peaks</td>
</tr>
<tr>
<td><a href="called_peaks_file/">called_peaks_file</a></td>
<td>name and path of the bed file with open chromatin peaks after QC filtering</td>
</tr>
<tr>
<td><a href="called_variants_dna_file/">called_variants_dna_file</a></td>
<td>name and path of the file with variant calls</td>
</tr>
<tr>
<td><a href="called_variants_dna_short_read_id/">called_variants_dna_short_read_id</a></td>
<td>unique key for table (anvil requirement)</td>
</tr>
<tr>
<td><a href="called_variants_nanopore_id/">called_variants_nanopore_id</a></td>
<td>unique key for table (anvil requirement)</td>
</tr>
<tr>
<td><a href="called_variants_pac_bio_id/">called_variants_pac_bio_id</a></td>
<td>unique key for table (anvil requirement)</td>
</tr>
<tr>
<td><a href="caller_software/">caller_software</a></td>
<td>variant calling software used including version number</td>
</tr>
<tr>
<td><a href="chemistry_type/">chemistry_type</a></td>
<td>chemistry type used for the experiment</td>
</tr>
<tr>
<td><a href="chrom/">chrom</a></td>
<td>Chromosome of the variant</td>
</tr>
<tr>
<td><a href="chrom_end/">chrom_end</a></td>
<td>End position chromosome of SV</td>
</tr>
<tr>
<td><a href="ClinGen_allele_ID/">ClinGen_allele_ID</a></td>
<td>ClinGen Allele ID for cross table refrence</td>
</tr>
<tr>
<td><a href="condition_id/">condition_id</a></td>
<td>MONDO/OMIM number for condition used for variant interpretation</td>
</tr>
<tr>
<td><a href="condition_inheritance/">condition_inheritance</a></td>
<td>Description of the expected inheritance of condition used for variant interpr...</td>
</tr>
<tr>
<td><a href="consanguinity/">consanguinity</a></td>
<td>Indicate if consanguinity is present or suspected within a family</td>
</tr>
<tr>
<td><a href="consanguinity_detail/">consanguinity_detail</a></td>
<td>Free text description of any additional consanguinity details</td>
</tr>
<tr>
<td><a href="consent_code/">consent_code</a></td>
<td>Consent group pertaining to this participant's data</td>
</tr>
<tr>
<td><a href="contamination/">contamination</a></td>
<td>Contamination level estimate</td>
</tr>
<tr>
<td><a href="copy_number/">copy_number</a></td>
<td>CNV copy number</td>
</tr>
<tr>
<td><a href="date_data_generation/">date_data_generation</a></td>
<td>Date of data generation (First sequencing date)</td>
</tr>
<tr>
<td><a href="estimated_library_size/">estimated_library_size</a></td>
<td></td>
</tr>
<tr>
<td><a href="experiment_atac_short_read_id/">experiment_atac_short_read_id</a></td>
<td>identifier for experiment</td>
</tr>
<tr>
<td><a href="experiment_dna_short_read_id/">experiment_dna_short_read_id</a></td>
<td>identifier for experiment</td>
</tr>
<tr>
<td><a href="experiment_id/">experiment_id</a></td>
<td>table_name</td>
</tr>
<tr>
<td><a href="experiment_nanopore_id/">experiment_nanopore_id</a></td>
<td>identifier for experiment</td>
</tr>
<tr>
<td><a href="experiment_pac_bio_id/">experiment_pac_bio_id</a></td>
<td>identifier for experiment</td>
</tr>
<tr>
<td><a href="experiment_rna_short_read_id/">experiment_rna_short_read_id</a></td>
<td>identifier for experiment</td>
</tr>
<tr>
<td><a href="experiment_sample_id/">experiment_sample_id</a></td>
<td>identifier used in the data file (e</td>
</tr>
<tr>
<td><a href="experiment_type/">experiment_type</a></td>
<td></td>
</tr>
<tr>
<td><a href="family_history_detail/">family_history_detail</a></td>
<td>Details about family history that do not fit into structured fields</td>
</tr>
<tr>
<td><a href="family_id/">family_id</a></td>
<td>Identifier for family (primary key)</td>
</tr>
<tr>
<td><a href="fragmentation_method/">fragmentation_method</a></td>
<td>method used for shearing/fragmentation</td>
</tr>
<tr>
<td><a href="gene_annotation/">gene_annotation</a></td>
<td>annotation file used for alignment</td>
</tr>
<tr>
<td><a href="gene_annotation_details/">gene_annotation_details</a></td>
<td></td>
</tr>
<tr>
<td><a href="gene_disease_validity/">gene_disease_validity</a></td>
<td>Validity assessment of the gene-disease relationship</td>
</tr>
<tr>
<td><a href="gene_known_for_phenotype/">gene_known_for_phenotype</a></td>
<td>Indicate if the gene listed is a candidate or known disease gene</td>
</tr>
<tr>
<td><a href="gene_of_interest/">gene_of_interest</a></td>
<td>HGNC approved symbol of the known or candidate gene(s) that are relevant for ...</td>
</tr>
<tr>
<td><a href="genetic_findings_id/">genetic_findings_id</a></td>
<td>Unique ID of this variant in this participant (primary key)</td>
</tr>
<tr>
<td><a href="genome_coverage/">genome_coverage</a></td>
<td>e</td>
</tr>
<tr>
<td><a href="gregor_center/">gregor_center</a></td>
<td>GREGoR Center to which the participant is originally associated</td>
</tr>
<tr>
<td><a href="GREGoR_ClinVar_SCV/">GREGoR_ClinVar_SCV</a></td>
<td>ClinVar accession number for the variant curation submitted by your center</td>
</tr>
<tr>
<td><a href="GREGoR_variant_classification/">GREGoR_variant_classification</a></td>
<td>Clinical significance of variant described to condition listed as determined ...</td>
</tr>
<tr>
<td><a href="het_sites_file/">het_sites_file</a></td>
<td>VCF file containing prefiltered heterozygous sites used for reference alignme...</td>
</tr>
<tr>
<td><a href="het_sites_md5sum/">het_sites_md5sum</a></td>
<td>md5 checksum for het_sites_file</td>
</tr>
<tr>
<td><a href="hgvs/">hgvs</a></td>
<td>genomic HGVS description of the variant</td>
</tr>
<tr>
<td><a href="hgvsc/">hgvsc</a></td>
<td>HGVS c</td>
</tr>
<tr>
<td><a href="hgvsp/">hgvsp</a></td>
<td>HGVS p</td>
</tr>
<tr>
<td><a href="hours_since_last_meal/">hours_since_last_meal</a></td>
<td>his is relevant when analyzing metabolomics data</td>
</tr>
<tr>
<td><a href="id_in_table/">id_in_table</a></td>
<td></td>
</tr>
<tr>
<td><a href="includes_CpG_methylation/">includes_CpG_methylation</a></td>
<td>run reports CpG methylation</td>
</tr>
<tr>
<td><a href="includes_kinetics/">includes_kinetics</a></td>
<td>run reports base kinetics</td>
</tr>
<tr>
<td><a href="instrument_ics_version/">instrument_ics_version</a></td>
<td>Version number of PacBio instrument control software</td>
</tr>
<tr>
<td><a href="internal_project_id/">internal_project_id</a></td>
<td>An identifier used by GREGoR research centers to identify a set of participan...</td>
</tr>
<tr>
<td><a href="known_condition_name/">known_condition_name</a></td>
<td>Free text of condition name</td>
</tr>
<tr>
<td><a href="library_prep_type/">library_prep_type</a></td>
<td>type of library prep</td>
</tr>
<tr>
<td><a href="library_size/">library_size</a></td>
<td>library prep - expected size of library from FemtoPulse</td>
</tr>
<tr>
<td><a href="linked_variant/">linked_variant</a></td>
<td>Second variant in recessive cases</td>
</tr>
<tr>
<td><a href="linked_variant_phase/">linked_variant_phase</a></td>
<td></td>
</tr>
<tr>
<td><a href="mapped_reads_pct/">mapped_reads_pct</a></td>
<td>Number between 1 and 100, na</td>
</tr>
<tr>
<td><a href="maternal_id/">maternal_id</a></td>
<td>participant_id for mother; 0 if not available</td>
</tr>
<tr>
<td><a href="md5sum/">md5sum</a></td>
<td>md5 checksum for file</td>
</tr>
<tr>
<td><a href="mean_coverage/">mean_coverage</a></td>
<td>Mean coverage of either the genome or the targeted regions</td>
</tr>
<tr>
<td><a href="method_of_discovery/">method_of_discovery</a></td>
<td>The method/assay(s) used to identify the candidate</td>
</tr>
<tr>
<td><a href="methylation_called/">methylation_called</a></td>
<td>Indicates whether 5mC methylation has been called and annotated in the BAM fi...</td>
</tr>
<tr>
<td><a href="missing_variant_case/">missing_variant_case</a></td>
<td>Indication of whether this is known to be a missing variant case, see notes f...</td>
</tr>
<tr>
<td><a href="missing_variant_details/">missing_variant_details</a></td>
<td>For missing variant cases, indicate gene(s) or region of interest and reason ...</td>
</tr>
<tr>
<td><a href="movie_length_hours/">movie_length_hours</a></td>
<td>sequencing - length of sequencing collection, in hrs</td>
</tr>
<tr>
<td><a href="movie_name/">movie_name</a></td>
<td>sequencing - unique name of sequencing collection</td>
</tr>
<tr>
<td><a href="notes/">notes</a></td>
<td>Free text field to explain edge cases or discovery updates or list parallel e...</td>
</tr>
<tr>
<td><a href="num_aligned_bases/">num_aligned_bases</a></td>
<td>Number of bases in aligned reads</td>
</tr>
<tr>
<td><a href="num_aligned_reads/">num_aligned_reads</a></td>
<td>Total aligned reads</td>
</tr>
<tr>
<td><a href="num_bases/">num_bases</a></td>
<td>Number of bases (before/ignoring alignment)</td>
</tr>
<tr>
<td><a href="num_reads/">num_reads</a></td>
<td>Total reads (before/ignoring alignment)</td>
</tr>
<tr>
<td><a href="onset_age_range/">onset_age_range</a></td>
<td></td>
</tr>
<tr>
<td><a href="ontology/">ontology</a></td>
<td></td>
</tr>
<tr>
<td><a href="partial_contribution_explained/">partial_contribution_explained</a></td>
<td>List of specific phenotypes (HPO IDs) explained by the condition associated w...</td>
</tr>
<tr>
<td><a href="participant_drugs_intake/">participant_drugs_intake</a></td>
<td>The list of drugs patient is on, at the time of sample collection</td>
</tr>
<tr>
<td><a href="participant_id/">participant_id</a></td>
<td></td>
</tr>
<tr>
<td><a href="participant_special_diet/">participant_special_diet</a></td>
<td>If the patient was fasting, when the sample was collected</td>
</tr>
<tr>
<td><a href="passage_number/">passage_number</a></td>
<td>passage_number is relevant for fibroblast cultures and possibly iPSC</td>
</tr>
<tr>
<td><a href="paternal_id/">paternal_id</a></td>
<td>participant_id for father; 0 if not available</td>
</tr>
<tr>
<td><a href="peak_caller_software/">peak_caller_software</a></td>
<td>peak calling software used including version number</td>
</tr>
<tr>
<td><a href="peak_set_type/">peak_set_type</a></td>
<td>peak set type, according to ENCODE descriptors</td>
</tr>
<tr>
<td><a href="peaks_md5sum/">peaks_md5sum</a></td>
<td>md5 checksum for called_peaks_file</td>
</tr>
<tr>
<td><a href="pedigree_file/">pedigree_file</a></td>
<td>name of file (renamed from pedigree_image because it can contain a PED file o...</td>
</tr>
<tr>
<td><a href="pedigree_file_detail/">pedigree_file_detail</a></td>
<td>Free text description of other family structure/pedigree file caption or lege...</td>
</tr>
<tr>
<td><a href="percent_chrX_Y/">percent_chrX_Y</a></td>
<td></td>
</tr>
<tr>
<td><a href="percent_GC/">percent_GC</a></td>
<td></td>
</tr>
<tr>
<td><a href="percent_Globin/">percent_Globin</a></td>
<td></td>
</tr>
<tr>
<td><a href="percent_mRNA/">percent_mRNA</a></td>
<td></td>
</tr>
<tr>
<td><a href="percent_mtRNA/">percent_mtRNA</a></td>
<td></td>
</tr>
<tr>
<td><a href="percent_multimapped/">percent_multimapped</a></td>
<td>how many reads aligned to multiple places</td>
</tr>
<tr>
<td><a href="percent_rRNA/">percent_rRNA</a></td>
<td></td>
</tr>
<tr>
<td><a href="percent_UMI/">percent_UMI</a></td>
<td></td>
</tr>
<tr>
<td><a href="percent_unaligned/">percent_unaligned</a></td>
<td>how many reads didn't align</td>
</tr>
<tr>
<td><a href="percent_uniquely_aligned/">percent_uniquely_aligned</a></td>
<td>how many reads aligned to just one place</td>
</tr>
<tr>
<td><a href="phenotype_contribution/">phenotype_contribution</a></td>
<td>Contribution of variant-linked condition to participant's phenotype</td>
</tr>
<tr>
<td><a href="phenotype_description/">phenotype_description</a></td>
<td>human-readable 'Phenotypic one-line summary' for why this individual is of in...</td>
</tr>
<tr>
<td><a href="phenotype_id/">phenotype_id</a></td>
<td>primary key</td>
</tr>
<tr>
<td><a href="pmid_id/">pmid_id</a></td>
<td>Case specific PubMed ID if applicable</td>
</tr>
<tr>
<td><a href="polymerase_kit/">polymerase_kit</a></td>
<td>sequencing - part number of polymerase kit used</td>
</tr>
<tr>
<td><a href="pos/">pos</a></td>
<td>Start position of the variant</td>
</tr>
<tr>
<td><a href="pos_end/">pos_end</a></td>
<td>End position of SV</td>
</tr>
<tr>
<td><a href="presence/">presence</a></td>
<td></td>
</tr>
<tr>
<td><a href="primary_biosample/">primary_biosample</a></td>
<td>Tissue type of biosample taken from the participant that the analyte was extr...</td>
</tr>
<tr>
<td><a href="primary_biosample_details/">primary_biosample_details</a></td>
<td>Free text to capture information not in structured fields</td>
</tr>
<tr>
<td><a href="primary_biosample_id/">primary_biosample_id</a></td>
<td>Optional ID for the biosample; allows for linking of multiple analytes extrac...</td>
</tr>
<tr>
<td><a href="prior_testing/">prior_testing</a></td>
<td>Text description of any genetic testing for individual conducted prior to enr...</td>
</tr>
<tr>
<td><a href="proband_relationship/">proband_relationship</a></td>
<td>Text description of individual relationship to proband in family, especially ...</td>
</tr>
<tr>
<td><a href="proband_relationship_detail/">proband_relationship_detail</a></td>
<td>Other proband relationship not captured in enumeration above</td>
</tr>
<tr>
<td><a href="public_database_ID_other/">public_database_ID_other</a></td>
<td>Public database variant/case ID</td>
</tr>
<tr>
<td><a href="public_database_other/">public_database_other</a></td>
<td>Public databases that this variant in this participant has been submitted by ...</td>
</tr>
<tr>
<td><a href="quality_issues/">quality_issues</a></td>
<td>describe if there are any QC issues that would be important to note</td>
</tr>
<tr>
<td><a href="read_error_rate/">read_error_rate</a></td>
<td>Mean empirical per-base error rate of aligned reads</td>
</tr>
<tr>
<td><a href="read_length/">read_length</a></td>
<td>sequenced read length (bp); GREGoR RCs do paired end sequencing, so is the ex...</td>
</tr>
<tr>
<td><a href="read_length_mean/">read_length_mean</a></td>
<td>Mean length of all reads (before/ignoring alignment)</td>
</tr>
<tr>
<td><a href="recontactable/">recontactable</a></td>
<td>Is the originating GREGoR Center likely able to recontact this participant</td>
</tr>
<tr>
<td><a href="ref/">ref</a></td>
<td>Reference allele of the variant</td>
</tr>
<tr>
<td><a href="reference_assembly/">reference_assembly</a></td>
<td></td>
</tr>
<tr>
<td><a href="reference_assembly_details/">reference_assembly_details</a></td>
<td></td>
</tr>
<tr>
<td><a href="reference_assembly_uri/">reference_assembly_uri</a></td>
<td></td>
</tr>
<tr>
<td><a href="reported_ethnicity/">reported_ethnicity</a></td>
<td>Self/submitter-reported ethnicity (OMB categories)</td>
</tr>
<tr>
<td><a href="reported_race/">reported_race</a></td>
<td>Self/submitter-reported race (OMB categories)</td>
</tr>
<tr>
<td><a href="RIN/">RIN</a></td>
<td>RIN number for quality of sample</td>
</tr>
<tr>
<td><a href="sample_transformation_detail/">sample_transformation_detail</a></td>
<td>details regarding sample transformation</td>
</tr>
<tr>
<td><a href="seq_library_prep_kit_method/">seq_library_prep_kit_method</a></td>
<td>Library prep kit used</td>
</tr>
<tr>
<td><a href="sequencing_event_details/">sequencing_event_details</a></td>
<td>describe if there are any sequencing-specific issues that would be important ...</td>
</tr>
<tr>
<td><a href="sequencing_kit/">sequencing_kit</a></td>
<td>sequencing - part number of sequencing kit reagents</td>
</tr>
<tr>
<td><a href="sequencing_platform/">sequencing_platform</a></td>
<td>sequencing platform used for the experiment</td>
</tr>
<tr>
<td><a href="sex/">sex</a></td>
<td>Biological sex assigned at birth (aligned with All of Us)</td>
</tr>
<tr>
<td><a href="sex_concordance/">sex_concordance</a></td>
<td>Comparison between reported sex vs genotype sex; Other if ploidy NOT XX or XY...</td>
</tr>
<tr>
<td><a href="sex_detail/">sex_detail</a></td>
<td>Optional free-text field to describe known discrepancies between 'sex' value ...</td>
</tr>
<tr>
<td><a href="single_or_paired_ends/">single_or_paired_ends</a></td>
<td>single or paired end</td>
</tr>
<tr>
<td><a href="size_selection_method/">size_selection_method</a></td>
<td>library prep - method use for library size selection</td>
</tr>
<tr>
<td><a href="smrt_cell_id/">smrt_cell_id</a></td>
<td>sequencing - unique serial number for SMRT Cell</td>
</tr>
<tr>
<td><a href="smrt_cell_kit/">smrt_cell_kit</a></td>
<td>sequencing - part number of the SMRT Cell</td>
</tr>
<tr>
<td><a href="smrtlink_server_version/">smrtlink_server_version</a></td>
<td>Version number of PacBio SMRTLink software</td>
</tr>
<tr>
<td><a href="solve_status/">solve_status</a></td>
<td>Indication of whether the submitting RC considers this case 'solved'</td>
</tr>
<tr>
<td><a href="sv_type/">sv_type</a></td>
<td></td>
</tr>
<tr>
<td><a href="syndromic/">syndromic</a></td>
<td>For participants with few HPO terms, this optional field is to provide contex...</td>
</tr>
<tr>
<td><a href="table_name/">table_name</a></td>
<td></td>
</tr>
<tr>
<td><a href="target_insert_size/">target_insert_size</a></td>
<td>insert size the protocol targets for DNA fragments</td>
</tr>
<tr>
<td><a href="targeted_region_bed_file/">targeted_region_bed_file</a></td>
<td>name and path of bed file uploaded to workspace</td>
</tr>
<tr>
<td><a href="targeted_regions_method/">targeted_regions_method</a></td>
<td>Which capture kit is used</td>
</tr>
<tr>
<td><a href="term_id/">term_id</a></td>
<td></td>
</tr>
<tr>
<td><a href="time_to_freeze/">time_to_freeze</a></td>
<td>time (in hours) from collection to freezing the sample</td>
</tr>
<tr>
<td><a href="tissue_affected_status/">tissue_affected_status</a></td>
<td>If applicable to disease (suspected mosaic), is the tissue from an affected s...</td>
</tr>
<tr>
<td><a href="total_reads/">total_reads</a></td>
<td>total number of reads</td>
</tr>
<tr>
<td><a href="transcript/">transcript</a></td>
<td>Text description of transcript overlapping the variant</td>
</tr>
<tr>
<td><a href="twin_id/">twin_id</a></td>
<td>participant_id for twins, triplets, etc; 0 if not available</td>
</tr>
<tr>
<td><a href="variant_inheritance/">variant_inheritance</a></td>
<td>Detection of variant in parents</td>
</tr>
<tr>
<td><a href="variant_reference_assembly/">variant_reference_assembly</a></td>
<td>The genome build for identifying the variant position</td>
</tr>
<tr>
<td><a href="variant_type/">variant_type</a></td>
<td></td>
</tr>
<tr>
<td><a href="variant_types/">variant_types</a></td>
<td>types of variants called</td>
</tr>
<tr>
<td><a href="was_barcoded/">was_barcoded</a></td>
<td>indicates whether samples were barcoded on this flowcell</td>
</tr>
<tr>
<td><a href="within_site_batch_name/">within_site_batch_name</a></td>
<td>batch number for the site, important for future batch correction</td>
</tr>
<tr>
<td><a href="zygosity/">zygosity</a></td>
<td>Zygosity of variant</td>
</tr>
</tbody>
</table>
<h2 id="enumerations">Enumerations</h2>
<table>
<thead>
<tr>
<th>Enumeration</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="AdditionalModifiersEnum/">AdditionalModifiersEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="AffectedStatusEnum/">AffectedStatusEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="AnalyteTypeEnum/">AnalyteTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ApplicationKitEnum/">ApplicationKitEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ChemistryTypeEnum/">ChemistryTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ChromEndEnum/">ChromEndEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ChromEnum/">ChromEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ConditionInheritanceEnum/">ConditionInheritanceEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ConsanguinityEnum/">ConsanguinityEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ConsentCodeEnum/">ConsentCodeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ExperimentTypeEnum/">ExperimentTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="GeneAnnotationDetailsEnum/">GeneAnnotationDetailsEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="GeneDiseaseValidityEnum/">GeneDiseaseValidityEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="GeneKnownForPhenotypeEnum/">GeneKnownForPhenotypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="GregorCenterEnum/">GregorCenterEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="GregorVariantClassificationEnum/">GregorVariantClassificationEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="LibraryPrepTypeEnum/">LibraryPrepTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="LinkedVariantPhaseEnum/">LinkedVariantPhaseEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="MethodOfDiscoveryEnum/">MethodOfDiscoveryEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="MissingVariantCaseEnum/">MissingVariantCaseEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="OnsetAgeRangeEnum/">OnsetAgeRangeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="OntologyEnum/">OntologyEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="PeakSetTypeEnum/">PeakSetTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="PhenotypeContributionEnum/">PhenotypeContributionEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="PresenceEnum/">PresenceEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="PrimaryBiosampleEnum/">PrimaryBiosampleEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ProbandRelationshipEnum/">ProbandRelationshipEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="RecontactableEnum/">RecontactableEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ReferenceAssemblyEnum/">ReferenceAssemblyEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ReportedEthnicityEnum/">ReportedEthnicityEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ReportedRaceEnum/">ReportedRaceEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="SeqLibraryPrepKitMethodEnum/">SeqLibraryPrepKitMethodEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="SequencingPlatformEnum/">SequencingPlatformEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="SexEnum/">SexEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="SingleOrPairedEndsEnum/">SingleOrPairedEndsEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="SolveStatusEnum/">SolveStatusEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="SvTypeEnum/">SvTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="SyndromicEnum/">SyndromicEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="TableNameEnum/">TableNameEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="TissueAffectedStatusEnum/">TissueAffectedStatusEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="VariantInheritanceEnum/">VariantInheritanceEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="VariantReferenceAssemblyEnum/">VariantReferenceAssemblyEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="VariantTypeEnum/">VariantTypeEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="VariantTypesEnum/">VariantTypesEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="ZygosityEnum/">ZygosityEnum</a></td>
<td></td>
</tr>
</tbody>
</table>
<h2 id="types">Types</h2>
<table>
<thead>
<tr>
<th>Type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Boolean/">Boolean</a></td>
<td>A binary (true or false) value</td>
</tr>
<tr>
<td><a href="Curie/">Curie</a></td>
<td>a compact URI</td>
</tr>
<tr>
<td><a href="Date/">Date</a></td>
<td>a date (year, month and day) in an idealized calendar</td>
</tr>
<tr>
<td><a href="DateOrDatetime/">DateOrDatetime</a></td>
<td>Either a date or a datetime</td>
</tr>
<tr>
<td><a href="Datetime/">Datetime</a></td>
<td>The combination of a date and time</td>
</tr>
<tr>
<td><a href="Decimal/">Decimal</a></td>
<td>A real number with arbitrary precision that conforms to the xsd:decimal speci...</td>
</tr>
<tr>
<td><a href="Double/">Double</a></td>
<td>A real number that conforms to the xsd:double specification</td>
</tr>
<tr>
<td><a href="Float/">Float</a></td>
<td>A real number that conforms to the xsd:float specification</td>
</tr>
<tr>
<td><a href="Integer/">Integer</a></td>
<td>An integer</td>
</tr>
<tr>
<td><a href="Jsonpath/">Jsonpath</a></td>
<td>A string encoding a JSON Path</td>
</tr>
<tr>
<td><a href="Jsonpointer/">Jsonpointer</a></td>
<td>A string encoding a JSON Pointer</td>
</tr>
<tr>
<td><a href="Ncname/">Ncname</a></td>
<td>Prefix part of CURIE</td>
</tr>
<tr>
<td><a href="Nodeidentifier/">Nodeidentifier</a></td>
<td>A URI, CURIE or BNODE that represents a node in a model</td>
</tr>
<tr>
<td><a href="Objectidentifier/">Objectidentifier</a></td>
<td>A URI or CURIE that represents an object in the model</td>
</tr>
<tr>
<td><a href="Sparqlpath/">Sparqlpath</a></td>
<td>A string encoding a SPARQL Property Path</td>
</tr>
<tr>
<td><a href="String/">String</a></td>
<td>A character string</td>
</tr>
<tr>
<td><a href="Time/">Time</a></td>
<td>A time object represents a (local) time of day, independent of any particular...</td>
</tr>
<tr>
<td><a href="Uri/">Uri</a></td>
<td>a complete URI</td>
</tr>
<tr>
<td><a href="Uriorcurie/">Uriorcurie</a></td>
<td>a URI or a CURIE</td>
</tr>
</tbody>
</table>
<h2 id="subsets">Subsets</h2>
<table>
<thead>
<tr>
<th>Subset</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
<td></td>
</tr>
</tbody>
</table>





                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

        </div>
        
      </main>
      
        <footer class="md-footer">
  
    
    <nav class="md-footer__inner md-grid" aria-label="Footer" >
      
      
        
        <a href="about/" class="md-footer__link md-footer__link--next" aria-label="Next: About" rel="next">
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Next
              </span>
              About
            </div>
          </div>
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4Z"/></svg>
          </div>
        </a>
      
    </nav>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": ".", "features": ["content.tabs.link"], "search": "assets/javascripts/workers/search.16e2a7d4.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="assets/javascripts/bundle.5a2dcb6a.min.js"></script>
      
    
    
  </body>
</html>