
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://madanucd.github.io/gregor_linkml/AlignedNanopore/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.6.0, mkdocs-material-8.5.10" name="generator"/>
<title>Class: AlignedNanopore - gregor_linkml</title>
<link href="../assets/stylesheets/main.975780f9.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.2505c338.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
</head>
<body data-md-color-accent="None" data-md-color-primary="None" data-md-color-scheme="default" dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#class-alignednanopore">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="gregor_linkml" class="md-header__button md-logo" data-md-component="logo" href=".." title="gregor_linkml">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            gregor_linkml
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Class: AlignedNanopore
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" title="Clear" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"></path></svg>
</button>
</nav>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/madanucd/gregor_linkml" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="gregor_linkml" class="md-nav__button md-logo" data-md-component="logo" href=".." title="gregor_linkml">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
    gregor_linkml
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/madanucd/gregor_linkml" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../about/">
        About
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#slots">
    Slots
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#identifier-and-mapping-information">
    Identifier and Mapping Information
  </a>
<nav aria-label="Identifier and Mapping Information" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#schema-source">
    Schema Source
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mappings">
    Mappings
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#linkml-source">
    LinkML Source
  </a>
<nav aria-label="LinkML Source" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#direct">
    Direct
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#induced">
    Induced
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/madanucd/gregor_linkml/edit/master/docs/AlignedNanopore.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"></path></svg>
</a>
<h1 id="class-alignednanopore">Class: AlignedNanopore</h1>
<p><em>Table containing aligned_nanopore information</em></p>
<p>URI: <a href="https://w3id.org/madanucd/gregor_linkml/AlignedNanopore">gregor_linkml:AlignedNanopore</a></p>
<div class="mermaid"> classDiagram
    class AlignedNanopore
    click AlignedNanopore href "../AlignedNanopore"
      AlignedNanopore : aligned_nanopore_file

      AlignedNanopore : aligned_nanopore_id

      AlignedNanopore : aligned_nanopore_index_file

      AlignedNanopore : aligned_read_length_mean

      AlignedNanopore : alignment_software

      AlignedNanopore : analysis_details

      AlignedNanopore : contamination

      AlignedNanopore : experiment_nanopore_id

      AlignedNanopore : genome_coverage

      AlignedNanopore : mapped_reads_pct

      AlignedNanopore : md5sum

      AlignedNanopore : mean_coverage

      AlignedNanopore : methylation_called

      AlignedNanopore : num_aligned_bases

      AlignedNanopore : num_aligned_reads

      AlignedNanopore : num_bases

      AlignedNanopore : num_reads

      AlignedNanopore : quality_issues

      AlignedNanopore : read_error_rate

      AlignedNanopore : read_length_mean

      AlignedNanopore : reference_assembly




    AlignedNanopore --&gt; "1" ReferenceAssemblyEnum : reference_assembly
    click ReferenceAssemblyEnum href "../ReferenceAssemblyEnum"


      AlignedNanopore : sex_concordance


</div>
<!-- no inheritance hierarchy -->
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../aligned_nanopore_id/">aligned_nanopore_id</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../experiment_nanopore_id/">experiment_nanopore_id</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>identifier for experiment</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../aligned_nanopore_file/">aligned_nanopore_file</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>name and path of file with aligned reads</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../aligned_nanopore_index_file/">aligned_nanopore_index_file</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>name and path of index file corresponding to aligned reads file</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../md5sum/">md5sum</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>md5 checksum for file</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../reference_assembly/">reference_assembly</a></td>
<td>1 <br/> <a href="../ReferenceAssemblyEnum/">ReferenceAssemblyEnum</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../alignment_software/">alignment_software</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Software including version number</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../analysis_details/">analysis_details</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>brief description of the analysis pipeline used for producing the asc_file; p...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../mean_coverage/">mean_coverage</a></td>
<td>0..1 <br/> <a href="../Float/">Float</a></td>
<td>Mean coverage of either the genome or the targeted regions</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../genome_coverage/">genome_coverage</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>e</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../contamination/">contamination</a></td>
<td>0..1 <br/> <a href="../Float/">Float</a></td>
<td>Contamination level estimate</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sex_concordance/">sex_concordance</a></td>
<td>0..1 <br/> <a href="../Boolean/">Boolean</a></td>
<td>Comparison between reported sex vs genotype sex; Other if ploidy NOT XX or XY...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../num_reads/">num_reads</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Total reads (before/ignoring alignment)</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../num_bases/">num_bases</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Number of bases (before/ignoring alignment)</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../read_length_mean/">read_length_mean</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Mean length of all reads (before/ignoring alignment)</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../num_aligned_reads/">num_aligned_reads</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Total aligned reads</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../num_aligned_bases/">num_aligned_bases</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Number of bases in aligned reads</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../aligned_read_length_mean/">aligned_read_length_mean</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Mean length of aligned reads</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../read_error_rate/">read_error_rate</a></td>
<td>0..1 <br/> <a href="../Float/">Float</a></td>
<td>Mean empirical per-base error rate of aligned reads</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../mapped_reads_pct/">mapped_reads_pct</a></td>
<td>0..1 <br/> <a href="../Float/">Float</a></td>
<td>Number between 1 and 100, na</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../methylation_called/">methylation_called</a></td>
<td>1 <br/> <a href="../Boolean/">Boolean</a></td>
<td>Indicates whether 5mC methylation has been called and annotated in the BAM fi...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../quality_issues/">quality_issues</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>describe if there are any QC issues that would be important to note</td>
<td>direct</td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: https://w3id.org/madanucd/gregor_linkml</li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>gregor_linkml:AlignedNanopore</td>
</tr>
<tr>
<td>native</td>
<td>gregor_linkml:AlignedNanopore</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<!-- TODO: investigate https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx -->
<h3 id="direct">Direct</h3>
<details>
<pre><code class="language-yaml">name: Aligned_nanopore
description: Table containing aligned_nanopore information
from_schema: https://w3id.org/madanucd/gregor_linkml
slots:
- aligned_nanopore_id
- experiment_nanopore_id
- aligned_nanopore_file
- aligned_nanopore_index_file
- md5sum
- reference_assembly
- alignment_software
- analysis_details
- mean_coverage
- genome_coverage
- contamination
- sex_concordance
- num_reads
- num_bases
- read_length_mean
- num_aligned_reads
- num_aligned_bases
- aligned_read_length_mean
- read_error_rate
- mapped_reads_pct
- methylation_called
- quality_issues

</code></pre>
</details>
<h3 id="induced">Induced</h3>
<details>
<pre><code class="language-yaml">name: Aligned_nanopore
description: Table containing aligned_nanopore information
from_schema: https://w3id.org/madanucd/gregor_linkml
attributes:
  aligned_nanopore_id:
    name: aligned_nanopore_id
    description: ''
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: aligned_nanopore_id
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    - Aligned_nanopore_set
    range: string
    required: true
  experiment_nanopore_id:
    name: experiment_nanopore_id
    description: identifier for experiment
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: experiment_nanopore_id
    owner: Aligned_nanopore
    domain_of:
    - Experiment_nanopore
    - Aligned_nanopore
    range: string
    required: true
  aligned_nanopore_file:
    name: aligned_nanopore_file
    description: name and path of file with aligned reads
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: aligned_nanopore_file
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    range: string
    required: true
  aligned_nanopore_index_file:
    name: aligned_nanopore_index_file
    description: name and path of index file corresponding to aligned reads file
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: aligned_nanopore_index_file
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    range: string
    required: true
  md5sum:
    name: md5sum
    description: md5 checksum for file
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: md5sum
    owner: Aligned_nanopore
    domain_of:
    - Aligned_dna_short_read
    - Called_variants_dna_short_read
    - Aligned_rna_short_read
    - Aligned_nanopore
    - Called_variants_nanopore
    - Aligned_pac_bio
    - Called_variants_pac_bio
    - Aligned_atac_short_read
    range: string
    required: true
  reference_assembly:
    name: reference_assembly
    description: ''
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: reference_assembly
    owner: Aligned_nanopore
    domain_of:
    - Aligned_dna_short_read
    - Aligned_rna_short_read
    - Aligned_nanopore
    - Aligned_pac_bio
    - Aligned_atac_short_read
    range: Reference_assemblyEnum
    required: true
  alignment_software:
    name: alignment_software
    description: Software including version number
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: alignment_software
    owner: Aligned_nanopore
    domain_of:
    - Aligned_dna_short_read
    - Aligned_rna_short_read
    - Aligned_nanopore
    - Aligned_pac_bio
    - Aligned_atac_short_read
    range: string
    required: true
  analysis_details:
    name: analysis_details
    description: brief description of the analysis pipeline used for producing the
      asc_file; perhaps a link to something like a WDL file or github repository
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: analysis_details
    owner: Aligned_nanopore
    domain_of:
    - Aligned_dna_short_read
    - Called_variants_dna_short_read
    - Aligned_nanopore
    - Called_variants_nanopore
    - Aligned_pac_bio
    - Called_variants_pac_bio
    - Called_peaks_atac_short_read
    - Allele_specific_atac_short_read
    range: string
  mean_coverage:
    name: mean_coverage
    description: Mean coverage of either the genome or the targeted regions
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: mean_coverage
    owner: Aligned_nanopore
    domain_of:
    - Aligned_dna_short_read
    - Aligned_rna_short_read
    - Aligned_nanopore
    - Aligned_pac_bio
    - Aligned_atac_short_read
    range: float
  genome_coverage:
    name: genome_coverage
    description: e.g. â‰¥90% at 10x or 20x; per consortium decision
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: genome_coverage
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    - Aligned_pac_bio
    range: integer
  contamination:
    name: contamination
    description: Contamination level estimate., e.g. &lt;1% (display raw fraction not
      percent)
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: contamination
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    - Aligned_pac_bio
    range: float
  sex_concordance:
    name: sex_concordance
    description: Comparison between reported sex vs genotype sex; Other if ploidy
      NOT XX or XY and Other if sex at birth is not known, thus unable to perform
      sex concordance
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: sex_concordance
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    - Aligned_pac_bio
    range: boolean
  num_reads:
    name: num_reads
    description: Total reads (before/ignoring alignment)
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: num_reads
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    - Aligned_pac_bio
    range: integer
  num_bases:
    name: num_bases
    description: Number of bases (before/ignoring alignment)
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: num_bases
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    - Aligned_pac_bio
    range: integer
  read_length_mean:
    name: read_length_mean
    description: Mean length of all reads (before/ignoring alignment)
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: read_length_mean
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    - Aligned_pac_bio
    range: integer
  num_aligned_reads:
    name: num_aligned_reads
    description: Total aligned reads
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: num_aligned_reads
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    - Aligned_pac_bio
    range: integer
  num_aligned_bases:
    name: num_aligned_bases
    description: Number of bases in aligned reads
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: num_aligned_bases
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    - Aligned_pac_bio
    range: integer
  aligned_read_length_mean:
    name: aligned_read_length_mean
    description: Mean length of aligned reads
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: aligned_read_length_mean
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    - Aligned_pac_bio
    range: integer
  read_error_rate:
    name: read_error_rate
    description: Mean empirical per-base error rate of aligned reads
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: read_error_rate
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    - Aligned_pac_bio
    range: float
  mapped_reads_pct:
    name: mapped_reads_pct
    description: Number between 1 and 100, na
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: mapped_reads_pct
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    - Aligned_pac_bio
    range: float
  methylation_called:
    name: methylation_called
    description: Indicates whether 5mC methylation has been called and annotated in
      the BAM file's MM and ML tags
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: methylation_called
    owner: Aligned_nanopore
    domain_of:
    - Aligned_nanopore
    - Aligned_pac_bio
    range: boolean
    required: true
  quality_issues:
    name: quality_issues
    description: describe if there are any QC issues that would be important to note
    from_schema: https://w3id.org/madanucd/gregor_linkml
    rank: 1000
    alias: quality_issues
    owner: Aligned_nanopore
    domain_of:
    - Analyte
    - Aligned_dna_short_read
    - Aligned_rna_short_read
    - Aligned_nanopore
    range: string

</code></pre>
</details>
</article>
</div>
<script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link"], "search": "../assets/javascripts/workers/search.16e2a7d4.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
<script src="../assets/javascripts/bundle.5a2dcb6a.min.js"></script>
<script src="https://unpkg.com/mermaid@10.9.0/dist/mermaid.min.js"></script><script>mermaid.initialize({});</script></body>
</html>